dsl_test.go

```diff
@@ -750,6 +751,23 @@ var _ = Describe("MockDisplay", func() {
 			})
 		})
 
+		Context("Data race", func(){
+			It("detect", func() {
```

Comment:
How about:

```go
Context("Concurrent access to mock", func() {
    It("does not panic", ...
```

Reply:
Thanks, I'm having trouble with the name of the tests. :-)